docs/modules/ROOT/pages/includes/attributes.adoc

```diff
@@ -0,0 +1,3 @@
+:project-version: 2.7.0
```

Comment:
Should this version be updated each time I do a release?

Reply:
This is updated automatically from the templates/ directory, so you don't need to worry about it